anki/tags.py

```diff
@@ -139,13 +139,16 @@ def addToStr(self, addtags, tags):
         return self.join(self.canonify(currentTags))
 
     def remFromStr(self, deltags, tags):
-        "Delete tags if they don't exists."
+        "Delete tags if they exist."
+        def wildcard(pat, str):
+            return '*' in pat and re.search(
+                pat.replace('*', '.*'), str, re.IGNORECASE)
```

Comment:
I usually wouldn't, but Damien has used `str` in this way pretty regularly throughout the existing code. It was just an attempt to match his style - for better or for worse :-)